spack/local/repo.yaml

```diff
@@ -1,2 +1,3 @@
 repo:
-  namespace: 'local'
+  namespace: local
+  api: v2.0
```

Comment:
I am hoping that these changes will ensure we use the latest package API, but I think we will need to manually update this every so often moving forward. I have asked if there's a way to always use the latest version, and will make another PR if needed to add that.